### PR TITLE
Add Spry Servers hosting provider

### PR DESCRIPTION
**Pull request by tabp0le**

### CODE REVIEW

File: _data/hosting_providers.json [hunk 1]
```diff
@@ -119,6 +119,16 @@
...
   },
+  {
+    "name": "Spry Servers",
+    "link": "https://www.spryservers.net/",
+    "category": "full",
+    "tutorial": "",
+    "announcement": "",
+    "plan": "",
+    "reviewed": "2019.6.24",
+    "note": "Applies to all shared and shared reseller plans, including WordPress."
```
Comment (adferrand) [inline _data/hosting_providers.json]: Some missing words here, maybe?

Comment (tabp0le) [reply _data/hosting_providers.json]: Can you clarify please?

**Comment by ohemorange** [comment]
Hey @tabp0le, just to check -- in the plans described, there's no extra steps users need to take to turn on HTTPS? Is there an announcement somewhere stating that? Also, did you mean "shared and shared reseller" or "shared and reseller"? What about the "nextcloud hosting" product?

**Comment by tabp0le** [comment]
@ohemorange Thanks for the info. I've updated with clarification and a tutorial. There are no extra steps, unless the process fails to secure initially. I've included a tutorial for when this happens. I also reworded the included plans. The standard nextcloud hosting is just an account on an existing nextcloud instance at this time. I'll update that once we start offering automated custom nextcloud installs.

Thanks again! :)

**Pull request merged**